api/VRPose.json

```diff
@@ -0,0 +1,243 @@
+{
+  "version": "1.0.0",
+  "data": {
+    "api": {
+      "VRPose": {
```

Comment:
Same here, deprecated methods missing with a status object: `hasOrientation`, `hasPosition` and `timestamp`.